
> rest-express@1.0.0 dev
> concurrently "npm:dev:client" "npm:dev:server"

[dev:server] 
[dev:server] > rest-express@1.0.0 dev:server
[dev:server] > tsx server/index.ts
[dev:server] 
[dev:client] 
[dev:client] > rest-express@1.0.0 dev:client
[dev:client] > vite --host 0.0.0.0
[dev:client] 
[dev:client] 
[dev:client]   VITE v6.0.11  ready in 365 ms
[dev:client] 
[dev:client]   ➜  Local:   http://localhost:5173/
[dev:client]   ➜  Network: http://172.31.196.75:5173/
[dev:server] info: Server running at http://0.0.0.0:5000/ {"hostname":"0.0.0.0","port":5000,"service":"arm-platform","timestamp":"2025-01-28T15:55:22.821Z"}
[dev:server] 3:55:24 PM [vite] ws error:
[dev:server] RangeError: Invalid WebSocket frame: invalid status code 16210
[dev:server]     at Receiver.controlMessage (/home/runner/workspace/node_modules/vite/dist/node/chunks/dep-M1IYMR16.js:34512:30)
[dev:server]     at Receiver.getData (/home/runner/workspace/node_modules/vite/dist/node/chunks/dep-M1IYMR16.js:34363:12)
[dev:server]     at Receiver.startLoop (/home/runner/workspace/node_modules/vite/dist/node/chunks/dep-M1IYMR16.js:34053:16)
[dev:server]     at Receiver._write (/home/runner/workspace/node_modules/vite/dist/node/chunks/dep-M1IYMR16.js:33980:10)
[dev:server]     at writeOrBuffer (node:internal/streams/writable:572:12)
[dev:server]     at _write (node:internal/streams/writable:501:10)
[dev:server]     at Writable.write (node:internal/streams/writable:510:10)
[dev:server]     at Socket.socketOnData (/home/runner/workspace/node_modules/vite/dist/node/chunks/dep-M1IYMR16.js:37037:37)
[dev:server]     at Socket.emit (node:events:530:35)
[dev:server]     at addChunk (node:internal/streams/readable:561:12)
[dev:server]     at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
[dev:server]     at Readable.push (node:internal/streams/readable:392:5)
[dev:server]     at TCP.onStreamRead (node:internal/stream_base_commons:191:23)
[dev:server]     at TCP.callbackTrampoline (node:internal/async_hooks:130:17)
[dev:server] npm run dev:server exited with code 1